Packs/AutoFocus/Integrations/AutoFocusTagsFeed/AutoFocusTagsFeed.py

```diff
@@ -0,0 +1,509 @@
+            return results
+
+    page_num = 0
+    has_updates = True
+    while has_updates:
+        response = client.get_tags({'pageNum': page_num,
+                                    'pageSize': 200,
+                                    'sortBy': 'updated_at',
+                                    'order': 'desc'})
+        tags = response.get('tags', [])
+        for tag in tags:
+            update_time = tag.get('updated_at')
+            update_time = datetime.strptime(update_time, AF_TAGS_DATE_FORMAT).strftime(
+                DATE_FORMAT) if update_time else None
+            update_time = date_to_timestamp(update_time, DATE_FORMAT)
+            if update_time >= time_from_last_update:
+                list_of_all_updated_tags.append(
+                    {'public_tag_name': tag.get('public_tag_name')})
+            else:
+                has_updates = False
+                break
+        page_num += 1
+
+    # add only PAGE_SIZE tag_details to results, so we wont make to many calls to the api
+    list_index = 0
+    for tag in list_of_all_updated_tags:
+        if len(results) < PAGE_SIZE:
+            public_tag_name = tag.get('public_tag_name')
+            response = client.get_tag_details(public_tag_name)
+            results.append(response)
+            list_index += 1
+        else:
+            break
+    # delete from the list all tags that will be returned this fetch
+    list_of_all_updated_tags = list_of_all_updated_tags[list_index:]
+    # update integration context
+    context = get_integration_context()
+    context['tags_need_to_be_fetched'] = list_of_all_updated_tags
+    context['time_of_first_fetch'] = date_to_timestamp(datetime.now(), DATE_FORMAT)
+    set_integration_context(context)
+    return results
+
+
+def get_tag_class(tag_class: Optional[str], source: Optional[str]) -> Optional[str]:
+    """
+    Returns the tag class as demisto indicator type.
+    Args:
+        tag_class: tag class name
+        source: tag source
+    Returns:
+        The tag class as demisto indicator type, None if class is not specified.
+    """
+
+    if not tag_class:
+        return None
+    if (tag_class != 'malicious_behavior') or (tag_class == 'malicious_behavior' and source == 'Unit 42'):
+        return TAG_CLASS_TO_DEMISTO_TYPE.get(tag_class)
+    return None
+
+
+def get_tag_groups(tag_groups: list) -> list:
+    """
+    Returns the tag groups as a list of the groups names.
+    Args:
+        tag_groups: list of all groups
+    Returns:
+        The tag groups as a list of the groups names
+    """
+
+    # Tag_groups is a list of dictionaries, each contains a tag group name and its description
+    results = []
+    if len(tag_groups) > 0:
+        for group in tag_groups:
+            tag_group_name = group.get('tag_group_name', '')
+            if tag_group_name:
+                results.append(tag_group_name)
+    return results
+
+
+def get_fields(tag_details: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Lets improve the name of this method by being more specific - what fields do we want to get? 
Indicator fields? for what purpose?